akka-docs/rst/additional/faq.rst

```diff
@@ -25,6 +25,28 @@ Akka is also:
 * a town in Morocco
 * a near-earth asteroid
 
+Resources with Explicit Lifecycle
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Actors, ActorSystems, ActorMaterializers (for streams), all these types of objects bind
+resources that must be released explicitly. The reason is that Actors are meant to have
+a life of their own, existing independently of whether messages are currently en route
+to them. Therefore you should always make sure that for every creation of such an object
+you have a matching ``stop``, ``terminate``, or ``shutdown`` call implemented.
+
+In particular you typically want to bind such values to immutable references, i.e.
+``final ActorSystem system`` in Java or ``val system: ActorSystem`` in Scala.
+
+JVM application or Scala REPL “hanging”
```

Comment:
yeah, great one to add (do we want to mention it's using non daemon threads? I worded it a bit differently here http://stackoverflow.com/questions/35583739/why-does-this-simple-akka-streams-program-never-terminate/35584236#35584236 perhaps too much tech detail?)